src/App.css

```diff
@@ -1,3 +1,53 @@
 body {
+  background-color: #fafafa;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  padding-left: 20px;
+  color: #0893ff;
+  font-weight: 500;
+  background-color: #fff;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  color: black;
+  font-weight: 300;
+  font-size: 120%;
+}
+
+.nav-links > * {
+  margin-left: 10px;
+  color: black;
+  text-decoration: none;
+}
+
+#add-book-form {
+  padding: 10px 50px;
+}
+
+#add-book-form > input {
+  margin-right: 20px;
+  padding: 5px;
+}
+
+#add-book-form > button {
+  margin-right: 20px;
+  padding: 5px 10px;
+  background-color: #0290ff;
+  color: #fafafa;
+  border: none;
+}
+
+.booksul {
+  list-style-type: none;
+}
+
+.book {
   background-color: white;
+  margin: 10px 20px;
+  padding: 10px 10px;
 }
```

Comment:
- [ ] It would be better to follow the design for book cards too. ⚒️ 
- You should implement everything from the template but you don't need to implement their functionality(comment, edit, update, progress circle). 🎨 
- Don't forget to use the correct font families. 🔴 

**Yours**
![image](https://user-images.githubusercontent.com/79876959/145666461-83bafa4e-4605-4809-a795-d504d79906a9.png)

**Template**
![image](https://user-images.githubusercontent.com/79876959/145666473-664af879-6648-461d-b71c-25221cb5c7f5.png)